resources/ctia-default.properties

```diff
@@ -13,6 +13,8 @@ ctia.http.enabled=true
 ctia.http.port=3000
 ctia.http.access-control-allow-origin=http://(localhost|127.0.0.1)(:\\d+)?
 ctia.http.access-control-allow-methods=get,post,put,delete
+ctia.http.access-control-allow-headers=server,date,content-type,connection,x-total-hits,x-next,x-sort,etag
```

Comment:
CORS are from the client POV. Why we need to add server, date and connection to this list?

Reply:
also add x-previous